platform/commonUI/edit/res/templates/edit-action-buttons.html

```diff
@@ -20,11 +20,30 @@
  at runtime from the About dialog for additional information.
 -->
 <span ng-controller="EditActionController">
-    <span ng-repeat="currentAction in editActions">
+    <!-- If there's a single save action show a button, otherwise show a dropdown with all save actions. -->
+    <span ng-if="saveActions.length === 1">
+        <a class='s-button major {{saveActions[0].getMetadata().cssclass}}'
+           title='{{saveActions[0].getMetadata().name}}'
+           ng-click="saveActions[0].perform()">
+            <span class="title-label">{{saveActions[0].getMetadata().name}}</span>
+        </a>
+    </span>
+
+    <span ng-if="saveActions.length > 1">
+        <mct-control key="'menu-button'"
+                     structure="{
+                        options: saveActionsAsMenuOptions,
+                        click: saveActionMenuClickHandler,
+                        cssclass: 'btn-bar right icon-save no-label major'
+                     }">
+        </mct-control>
+    </span>
+
+    <span ng-repeat="currentAction in otherEditActions">
         <a class='s-button {{currentAction.getMetadata().cssclass}}'
```

Comment:
Created a [new issue](https://github.com/nasa/openmct/issues/1198) to fix this later.